### PR TITLE
fix: start moving color constants into their own file

### PR DESCRIPTION
**Pull request by beyackle**
## Description

This starts the work to be finished at a later time to streamline and simplify Composer's design language by pulling widely-used constants like colors, font sizes, and dimensions into their own authoritative files out in a "constants" directory. This particular PR just addresses some of the colors, with future work to be done on the rest of the colors as well as the other files with constants in them.

## Task Item

refs #8014 
refs #6873

## Screenshots

Warning and error icons are now a consistent red and yellow wherever they appear.
image

### CODE REVIEW

File: Composer/packages/client/src/components/ProjectTree/treeItem.tsx [hunk 1]
```diff
@@ -202,14 +204,14 @@ const diagnosticIcon = {
...
-  background: '#FED9CC',
+  color: colors.errorFg,
+  background: colors.errorBg,
 };
 
 const diagnosticWarningIcon = {
   ...diagnosticIcon,
-  color: '#8A8780',
-  background: '#FFF4CE',
+  color: colors.warningFg,
```
Comment (srinaath) [inline Composer/packages/client/src/components/ProjectTree/treeItem.tsx:213]: what does Fg mean?

File: Composer/packages/client/src/constants/colors.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import { getTheme } from 'office-ui-fabric-react/lib/Styling';
+
+const theme = getTheme();
+
+const semColors = theme.semanticColors;
+
+export const colors = {
+  // copied from https://developer.microsoft.com/en-us/fluentui#/styles/web/colors/messaging
+  errorBg: semColors.errorBackground,
```
Comment (srinaath) [inline Composer/packages/client/src/constants/colors.ts:12]: nit: I usually prefer expanded names. Bg and Fg im assuming are Background and foreground.

File: Composer/packages/client/src/components/ProjectTree/treeItem.tsx (hunk 1 above)
Comment (beyackle) [reply Composer/packages/client/src/components/ProjectTree/treeItem.tsx:213]: Foreground

**Comment by beyackle** [comment]
Closing this temporarily; will reopen or redo as needed in R15 when the designs are more settled.

**Pull request closed without merging**